Exercise_C/index.html

```diff
@@ -16,7 +16,12 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <div class="flex-container">
+      <div class="blue box"></div>
+      <div class="red box"></div>
+      <div class="green box"></div>
+    </div>
+  </div>
     <script src="" async defer></script>
   </body>
 </html>
```

Comment:
love how you have used an additional class named "box" to make it easier for you to target it in the css